polkadot/runtime/parachains/src/scheduler.rs

```diff
@@ -236,8 +235,16 @@ impl<T: Config> Pallet<T> {
 		if n_cores == 0 || validators.is_empty() {
 			ValidatorGroups::<T>::set(Vec::new());
 		} else {
-			let group_base_size = validators.len() / n_cores as usize;
-			let n_larger_groups = validators.len() % n_cores as usize;
+			let group_base_size =
+				validators.len().checked_div(n_cores as usize).unwrap_or_else(|| {
+					defensive!("n_cores should not be 0");
+					0
+				});
```

Comment:
```suggestion
				validators.len().checked_div(n_cores as usize).defensive_proof("n_cores should not be 0").unwrap_or(0);
```
Just need to import `frame_support::traits::Defensive`.

Reply:
Thanks! Didn't know about `defensive_proof`.